Events/Ch17.event

```diff
@@ -154,10 +155,10 @@ VillageEventItem(PlaceholderText, HouseBG, Spear)
 ENDA
 
 BeginningScene:
-LOAD2 0x1 Units
-ENUN
 LOAD1 0x1 NicholRecruit
 ENUN
+LOAD2 0x1 Units
```

Comment:
Remove the load of the units altogether (unless they play a role in the opening events). They are automatically taken care of by the prep screen.

Reply:
(Keep Nichol's load, though)